packages/api/tests/test-endpoints-providers.js

```diff
@@ -25,13 +28,16 @@ const keyId = 'public.pub';
 const hash = { name: 'id', type: 'S' };
 
 async function setup() {
-  await bootstrap.bootstrapElasticSearch('http://localhost:4571');
+  await bootstrap.bootstrapElasticSearch('http://localhost:4571', esIndex);
   await models.Manager.createTable(process.env.ProvidersTable, hash);
   await providers.create(testProvider);
 }
 
 async function teardown() {
   await models.Manager.deleteTable(process.env.ProvidersTable);
+
+  const esClient = await Search.es('http://localhost:4571');
```

Comment:
minor but suggest making `'http://localhost:4571'` a `const`

Reply:
This gets reset anyway when in test mode, but if you have null for host it won't do anything. I changed them all to 'fakehost' so there's no (well, less) confusion.